packages/i18n/src/locales/en_US.js

```diff
@@ -951,6 +951,12 @@ const translations = {
               example: '',
               parameters: {}
             },
+            drop: {
+              link: '',
```

Comment:
link: https://docs.mongodb.com/manual/reference/method/db.collection.drop
description: Removes a collection or view from the database.
example: db.students.drop()

Reply:
thanks!